### PR TITLE
Windows Fixes

### PR DESCRIPTION
**Pull request by ajayyy**
Currently the only way to close a window is CTRL+W, but it works fine. You should probably double check if it still works on macOS

### CODE REVIEW

File: app/docked-window.js [hunk 1]
```diff
@@ -100,6 +100,11 @@ class DockManager {
         var screenHeight = screenSize().height;
         var x = this._anchor.position(window._index);
         var y = screenHeight - window.height;
+		var os = require('os');
```
Comment (dhleong) [inline app/docked-window.js]: Please indent with 4-spaces instead of tabs

File: app/docked-window.js [hunk 2]
```diff
@@ -100,6 +100,11 @@ class DockManager {
...
+
+		if(os.type().startsWith("Windows")){
+			y -= 40
```
Comment (dhleong) [inline app/docked-window.js]: It would be better to use `os.platform() === 'win32'`, and declared up top as a constant.

Also, the style is a space after `if` and before the opening `{`, and end the line with a semicolon.

File: app/docked-window.js [hunk 3]
```diff
@@ -172,7 +177,8 @@ class DockedWindow {
         // Create the browser window.
         var win = this.win = new BrowserWindow({
             titleBarStyle: 'hidden-inset',
-
+			frame: false,
```
Comment (dhleong) [inline app/docked-window.js]: tabs -> spaces please

File: app/package.json [hunk 4]
```diff
@@ -1,9 +1,19 @@
 {
-  "name" : "hangr",
-  "version" : "0.0-DEV",
+  "name": "hangr",
```
Comment (dhleong) [inline app/package.json]: Please revert the changes to this file

File: release.py [hunk 5]
```diff
@@ -12,11 +12,11 @@
 try:
     from hostage import *
 except ImportError:
-    print "!! Release library unavailable."
-    print "!! Use `pip install hostage` to fix."
```
Comment (dhleong) [inline release.py]: Also, please revert the changes to this file.

**Comment by ajayyy** [comment]
Ok, sorry. Fixes will be done before I send another request.

File: app/docked-window.js (hunk 2 above)
Comment (ajayyy) [inline app/docked-window.js]: wow I missed a semi colon... That's pretty funny actually.

**Comment by dhleong** [comment]
Not a problem! You can just update this PR and add a comment.

**Comment by ajayyy** [comment]
Should be fixed to the code style, but with the last one, I reverted to what you currently have, and it still is like that, so I'm not sure what you mean.

**Comment by dhleong** [comment]
The `app/package.json` file should go back to the dummy values; it's just a stub that gets filled in by the build scripts. The real package.json is one directory higher.

**Comment by ajayyy** [comment]
Fixed

**Comment by ajayyy** [comment]
Also, just added one more commit to fix an error that popped up on windows when trying to hide the icon from the dock. (You can't do that on windows I guess)

**Comment by dhleong** [comment]
Hey sorry I've been busy and haven't had a chance to test this out, but it's still on my radar. I'm a little bit concerned that disabling the scrollbar with that CSS is going to *completely* disable them on macOS, which is not desirable, but I'll try to find time to test and get back to you.

**Comment by ajayyy** [comment]
That's ok, if it doesn't work. I could somehow implement platform specific css, but not 100% sure how that would work. Take your time, and thanks for your awesome work!

**Comment by dhleong** [comment]
Closing this in favor of #50. Thanks again for your help investigating these issues!
